### ISSUE TITLE
Vite virtual imports not recognized by Knip 

### ISSUE THREAD

**Issue by ollejernstrom** [webpro/knip#370]
When using [vite virual imports](https://vitejs.dev/guide/api-plugin#virtual-modules-convention) "virtual" appears as an unused dependency when running knip.

Example:
```ts
import { registerSW } from 'virtual:pwa-register'
```

Knip output:
```knip
Unlisted dependencies (1)
virtual  src/index.tsx
```

Actual package name: `vite-plugin-pwa`

**Comment by webpro** [comment]
I can we translate `virtual:pwa-register` to `vite-plugin-pwa`?

**Comment by ollejernstrom** [comment]
I am not saying we necessarily need to translate it, but maybe knip should not mark these as unused since it is very confusing for users. Either we should ignore them by default when using vite or at least make it clear when using the vite plugin (in the docs perhaps).

**Comment by webpro** [comment]
:rocket: _This issue has been resolved in v3.3.3. See [Release 3.3.3](https://github.com/webpro/knip/releases/tag/3.3.3) for release notes._

**Comment by webpro** [comment]
Knip no longer sanitizes `virtual:*` specifiers, so you seem them unaltered in the report, and you can then add them to `ignoreDependencies`.

Another option could be to ignore them altogether, but that would be harder to implement and it may lead to confusion (not sure as I know _virtually_ nothing about this technique).
